src/form-data-wrapper.js

```diff
@@ -47,10 +48,10 @@ export class FormDataWrapper {
    *     input content.
    */
   constructor(opt_form = undefined) {
-    /** @private @const {FormData} */
+    /** @private @const {!FormData} */
     this.formData_ = new FormData(opt_form);
 
-    /** @private @const {!Object|undefined} */
+    /** @private @const {!Object<string, !Array<string>>|undefined} */
```

Comment:
Minor: Removing `|undefined` will be easier to read. Use `{Object<string, !Array<string>>}` and default to `null` if entries exists instead. Object types are nullable by default.

Reply:
Done. I explicitly used the nullable specifier `?` though, as the style guide suggests:

> all types except primitives and record literals must be annotated explicitly with either ? or ! to indicate whether they are nullable or not.

https://google.github.io/styleguide/jsguide.html#jsdoc-nullability